### ISSUE TITLE
How to implement variable length data

### ISSUE THREAD

**Issue by KammutierSpule** [marcinbor85/cAT#48]
Is possible to receive variable length data after command?
Example:
```
AT+CMQTTTOPIC=0,11
>simcomtopic
OK
AT+CMQTTPAYLOAD=0,13
>simcompayload
OK
```

**Comment by marcinbor85** [comment]
No, this is example of two-stages command processing and such feature is not supported by cAT.

**Comment by KammutierSpule** [comment]
Do you think is possible to extend the code to support it? Any suggestion where to look at in the source code?

**Comment by marcinbor85** [comment]
Yes. It should be relative easy. Just add another internal state.

**Comment by KammutierSpule** [comment]
Ok thanks, I will need to investigate. Could you point me to some lines in the code where that can be added?
